src/openforms/contrib/brk/tests/base.py

```diff
@@ -19,6 +19,13 @@
     header_value=BRK_API_KEY,
 )
 
+INVALID_BRK_SERVICE = ServiceFactory.build(
+    api_root="https://api.brk.kadaster.nl/invalid",
+    oas="https://api.brk.kadaster.nl/esd-eto-apikey/bevragen/v2/",  # ignored/unused
```

Comment:
you can omit the `oas`, it's disabled through `settings.ZGW_CONSUMERS_IGNORE_OAS_FIELDS`